### ISSUE TITLE
Web Clipper Screenhot not working both with Chrome and Firefox

### ISSUE THREAD

**Issue by fedonr** [laurent22/joplin#8090]
Windows 11

Latest Build of Chrome and Firefox and Latest version of Joplin Web Clipper 

I can a dot in the right side section of the App where there should be a Screeshot

![Screenshot (29)](https://user-images.githubusercontent.com/120009841/234952211-7344e9b2-5407-4078-9cb0-bc0238ff8b9d.png)

**Comment by dpoulton-github** [comment]
(Copied from the linked forum [thread](https://discourse.joplinapp.org/t/joplin-web-clipper-issue-with-screenshot/30604))

Joplin v2.9.17
Firefox 112.0.2
Windows 11 21H2
Clipper 2.11.2

Using the above set-up and selecting screenshot from the webclipper successfully captures whatever screen area I then select using the mouse "click and drag". The only way I can simulate what you are seeing is if I just click on the screen (without dragging a capture area) and grab the single pixel under the mouse.

**Comment by fedonr** [comment]
Hey Awesome! It worked, so let me explain the issue here, so basically it isnt a bug, but mainly a confusion.

If we can replace the term "Drag and release" with something like Snip it, that would be great. As not only me, but the friends whom I showed this issue, were also new to Joplin, and they also though of it as Drag the part of the screen you like to Snip and release.

So we thought of it as more of a scrolling action rather than snipping action. So we Scrolled the Part of page we wanted to grab and released it, where going back to Jopling gave us the mighty ".".

While now that I tried Snipping the part of it, that worked great.

Closing it with the humble suggestion, if rather than "Drag and Release" if we can address it in a better way where screw heads like me wont go around thinking of it as Scrolling action, and snip it. THat would be great.

As not only me but 2 friends of mine also though of it as Scrolling action and not snipping, as they are new to Joplin as well.
